Lib9c/Policy/BlockPolicySource.Utils.cs

```diff
@@ -0,0 +1,380 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Blockchain;
+using Libplanet.Blocks;
+using Libplanet.Tx;
+using Nekoyume.Action;
+using Nekoyume.Model.State;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+
+namespace Nekoyume.BlockChain.Policy
+{
+    // Collection of helper methods not directly used as a pluggable component.
```

Comment:
IMO some of these helper methods can be `private`...

Reply:
Either most should be `private` or `public`, but I guess `internal` might not be right. Initially I thought each method should be tested separately, but didn't want to expose them as `public`. Then again, the implementation detail is not quite finalized, so didn't really want to write tests which would be mostly overwritten in near future. ☹️

Reply:
Eh... changing things to `private` with unused parameters gives me build errors... 😮